plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TrinoHiveCatalog.java

```diff
@@ -298,13 +303,51 @@ public Transaction newCreateTableTransaction(
     public void dropTable(ConnectorSession session, SchemaTableName schemaTableName, boolean purgeData)
     {
         // TODO: support path override in Iceberg table creation: https://github.com/trinodb/trino/issues/8861
-        Table table = loadTable(session, schemaTableName);
+        TableMetadata metadata = tableMetadataCache.computeIfAbsent(
+                schemaTableName,
+                ignore -> ((BaseTable) loadIcebergTable(metastore, tableOperationsProvider, session, schemaTableName)).operations().current());
+
+        Table table = getIcebergTableWithMetadata(metastore, tableOperationsProvider, session, schemaTableName, metadata);
```

Comment:
fix code layout after inline

```suggestion
        Table table = getIcebergTableWithMetadata(metastore, tableOperationsProvider, session, schemaTableName, metadata);

```

Reply:
@findepi the code snippets are the same. Can you please rewrite the snippet suggestion?

Reply:
@findinpath the difference is in where the empty line is positioned.
i see the github UI is not displaying this clearly

Reply:
Applied cosmetic changes on the method code.
I hope I did understand correctly what you meant.